redirects.yaml

```diff
@@ -0,0 +1 @@
+/install.exe: /install/v2.0.0.exe
```

Comment:
Is this due to the lack of latest version? It would be a shame to have to update this will each release.

Reply:
Yes, that's right. If they simply added `microk8s-installer.exe` to the actual latest release, then we could simply proxy `/install.txt` to `https://github.com/ubuntu/microk8s/releases/latest/download/microk8s-installer.exe` and everything would work, I think. No need for a redirect at all (or rather, the redirect is managed by GitHub for us). But for the time being, the only way to do this is for us to hardcode somewhere the latest version.

Reply:
I'd be very happy not to add this redirect and wait for them to make this change instead, but @pmahnke seemed to be saying this was urgent.

Reply:
I was told by the microk8s team:

> Thanks for looking into this.  We do this because the installer is separate from MicroK8s in terms of operation, but lives in the same repo.  We have discussed bificating this repo in the past, but decided against it to keep the code together; is this the only way it can work with the redirect?

Reply:
The answer is yes, it is the only way it can work with the redirect.

Well, I suppose they have 2 options - one is to create a separate repo for the installer and put releases of the installer on that repo. The other is to add the installer to every release of microk8s.

Either way, the installer file needs to be attached to a "latest" release for the redirect to work.

Reply:
Ok thanks

Reply:
how does this work as a proxy to github?  I don't see the final URL in here at all?

Reply:
oh, the deployment config